### PR TITLE
web: Split `StreamingProgress` styles into multiple css modules files and colocate them with subcomponents

### PR DESCRIPTION
**Pull request by gitstart-sourcegraph**
## Description
In process of migration from global CSS to CSS modules using  manual fixes

## Refs
- [SG Issue](https://github.com/sourcegraph/sourcegraph/issues/26498)
- [Gitstart Ticket](https://app.gitstart.com/clients/sourcegraph/tickets/SG-26498)

## Success Criteria

The stylesheets in the following directory are converted to CSS modules.

- client/web/src/search/results/progress

### CODE REVIEW

**Review by valerybugakov — COMMENTED** [review]
@gitstart-sourcegraph, is this PR ready for review? Asking because it's in the draft status.

**Comment by gitstart-sourcegraph** [comment]
> @gitstart-sourcegraph, is this PR ready for review? Asking because it's in the draft status.

@valerybugakov it's nearly done, in this file we are using `$icon-inline-size` which is from `client/shared/src/global-styles/icons.scss`, there is no css variable `--icon-inline-size` so we want to confirm that:

1.  we can add that variable `--icon-inline-size` in this file and use it in new components. 
2. Should we replace all usages of `$icon-inline-size` with `var(--icon-inline-size)`

**Review by valerybugakov — COMMENTED** [review]
Let's replace all usage instances with the CSS variable.

File: client/web/src/search/results/progress/StreamingProgressSkippedButton.module.scss [hunk 1]
```diff
@@ -0,0 +1,9 @@
+.skipped {
+    // TODO: Not sure why we have this, so disable temporarily
+    // margin-bottom: 0;
```
Comment (limitedmage) [inline client/web/src/search/results/progress/StreamingProgressSkippedButton.module.scss:3]: If this is not needed anymore, remove it.

**Review by limitedmage — APPROVED** [review]

**Review by valerybugakov — APPROVED** [review]

**Pull request merged**